tests/ci_tests/code_formatter/crewai/test_crewai_main.py

```diff
@@ -0,0 +1,453 @@
+"""Tests for coffee_maker/code_formatter/crewai/main.py"""
+
+from unittest import mock
```

Comment:
Line 3-3: Alphabetized imports.
```suggestion
import pytest
```